docs/introduction.rst

```diff
@@ -53,127 +56,143 @@ Survey of North America (BBS) and the Alwyn Gentry Tree Transect data:
+        help
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      -v, --version         show program's version number and exit
+      -q, --quiet           suppress command-line output
+
+To install datasets, use the ``install`` command.
+
+Examples
+~~~~~~~~
+
+**Using install**
+
+$ ``retriever install -h`` (gives install options)
+
+::
 
     usage: retriever install [-h] [--compile] [--debug]
-                             {mysql,postgres,sqlite,msaccess,csv} ...
+                             {mysql,postgres,sqlite,msaccess,csv,json,xml} ...
 
     positional arguments:
-      {mysql,postgres,sqlite,msaccess,csv}
+      {mysql,postgres,sqlite,msaccess,csv,json,xml}
                             engine-specific help
         mysql               MySQL
         postgres            PostgreSQL
         sqlite              SQLite
         msaccess            Microsoft Access
         csv                 CSV
+        json                JSON
+        xml                 XML
 
     optional arguments:
       -h, --help            show this help message and exit
       --compile             force re-compile of script before downloading
       --debug               run in debug mode
 
 
-Examples
-~~~~~~~~
+**Examples using install with engines**
 
 
+These examples are using Breeding Bird Survey data (Breed-bird-survey)
+Using a specific database engine. The retriever has support for various engines; mysql, postgres, sqlite, msaccess, csv, json, xml, download_only
 
-These examples are using Breeding Bird Survey data (BBS)
+Install data into Mysql::
 
-Using Install::
+   retriever install mysql –user myusername –password ***** –host localhost –port 8888 –database_name testdbase Breed-bird-survey
+   retriever install mysql -user (using attributes in the client authentication configuration file)
 
-   retriever install -h (gives install options)
+Install data into postgres::
 
-Using a specific database engine. The retriever has support for various engines; mysql, postgres, sqlite, msaccess, csv, download_only::
+   retriever install postgres –user myusername –password ***** –host localhost –port 5432 –database_name testdbase Breed-bird-survey
+   retriever install postgres Breed-bird-survey (using attributes in the client authentication configuration file)
 
-   retriever install {Engine}
+Install data into sqlite::
 
-   retriever install mysql -h     ..(gives install mysql options)::
+   retriever install sqlite Breed-bird-survey -f mydatabase.db (will use or create default mydatabase.db)
+   retriever install sqlite Breed-bird-survey (will use or create default sqlite.db in working directory)
 
-   retriever install mysql –user myuser –password ***** –host localhost –port 8888 –database_name testdbase BBS
+Install data into csv::
 
-install data into an sqlite database named mydatabase.db you would use::
+   retriever install csv Breed-bird-survey --table_name  BBS
```

Comment:
I think this might end up overwriting each table and result in only the last one existing as BBS, in which case we'd actually want something like `BBS_{table}.csv`. Is that right?